### ISSUE TITLE
Admin\User and Access Requests - Internal User can view operator prime admin access requests

### ISSUE THREAD

**Issue by andrea-williams** [bcgov/cas-registration#1984]
#### Dependencies

- Blocked by
- Blocking

#### Description:

Internal ("cas_admin" and "cas_analyst") users need the ability to view and approve/deny access requests for "prime admin" users (the first external user requesting access to an Operator).

We implemented this functionality already in Part 1 so some of the work can be reused, but there are some significant changes required on the frontend for Reg Part 2 because we're more explicitly separating the Users and Operators views now.

#### Acceptance Criteria:

Given I am an authenticated `cas_admin` or `cas_analyst` user
When I click on the "Operator Administrators and Access Requests" tile of the Administration module
Then I can view all operator admin ("prime admin") access requests from external users

Given I am an authenticated `cas_admin` or `cas_analyst` user
When I'm on the "Operator Administrators and Access Requests" tile of the Administration module
Then I can approve or deny any pending operator admin ("prime admin") access requests from external users

#### Additional Context:

- tile wireframe: https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7781-6399&t=856fUpLVL7lOsXQ6-1
- wireframe of page: needs wireframe @zoeyli-46 

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

**Comment by zoeyli-46** [comment]
Could we merge these 2 tickets? [#2059](https://github.com/bcgov/cas-registration/issues/2059) 2059 has the updated wireframes and also updated requirements

**Comment by zoeyli-46** [comment]
should we close this ticket since its captured in #2160 ? @andrea-williams @patriciarussellCAS

**Comment by patriciarussellCAS** [comment]
Thanks @zoeyli-46 - i think we can probably close this. If we have missed anything I think it will get caught in our user role testing ticket. Happy to close.

**Comment by zoeyli-46** [comment]
closed :)
